FSharpLu.Azure/AzureVM.fs

```diff
@@ -1031,18 +1031,19 @@ let tryGetManagedImageByResourceId (context: Context.InfrastructureContext) (ima
             if isNull image then
                 return None
             else
-                TraceTags.info "Found an image with matching name in the resource group." tags
+                TraceTags.info "Found a managed with matching name in the resource group." tags
```

Comment:
You probably want "Found a managed image with a matching name in the resource group"